### DISCUSSION TITLE
Mongoose Wizard, page with pagination

### DISCUSSION THREAD

**Discussion by beygaba** [cesanta/mongoose#2804] [Ask questions, get answers] [answered]
Hello! 
I'm trying to use the wizard - it's a really powerful tool! Although the code it generates does not compile and does not work - you need to add something to main.c :)
I did not find in the wizard the ability to add a page with events/notifications with pagination (as is implemented in the Device Web UI Dashboard example).
Please tell me how to add a similar page with pagination in the wizard?

**Comment by scaprile** [comment, marked as answer]
The code generated by our Wizard DOES compile and DOES work.
Maybe for YOU, in YOUR specific scenario, that you don't describe, for some reason that you don't tell, YOU can't make it compile, hence, code that does not build: can't work. Probably it is YOU that have to add something to YOUR main.c due to YOUR IDE, that you don't say which one it is; or perhaps YOUR hardware, that you also don't say which one it is.

Or maybe you are right, and under some combination there is a problem we didn't see, and we won't be able to see it unless you properly describe what you have, what you do, where, using what, and why it does not build (compilers issue error messages). So, did you see the Wizard info page ? At the very end of that page there is a sentence with a link. Please honor that sentence and follow that link, sending all your comments to the proper channel. This repository is about Mongoose Library.

Thank you.
